installation/resources/kyma/kyma-overrides-minimal.yaml

```diff
@@ -144,10 +155,40 @@ monitoring:
         requests:
           cpu: 50m
           memory: 64Mi
+  prometheus:
+    prometheusSpec:
+      alertingEndpoints:
+        - apiVersion: v2
+          name: monitoring-alertmanager
+          namespace: kyma-system
+          pathPrefix: /
+          port: http-web
+          scheme: https
+          # By default in Kyma, Prometheus injects the Istio certificates from a sidecar
+          tlsConfig:
+            caFile: /etc/prometheus/secrets/istio.default/root-cert.pem
+            certFile: /etc/prometheus/secrets/istio.default/cert-chain.pem
+            keyFile: /etc/prometheus/secrets/istio.default/key.pem
+            insecureSkipVerify: true
```

Comment:
You've adapted a part of the mTLS patches to be through helm and the other part to be through bash.
<img width="1561" alt="image" src="https://github.com/kyma-incubator/compass/assets/31803034/d70797c5-d3f4-47c4-a7a4-9d8626b029b6">

Is it possible for all of the patches to be in the helm charts? My concern is that the current patches are all in one place and with the new setup there are in a couple of places(with a couple of different overrides)? it's debatable but it could make the supportability harder?

P.S. Do prom dashboards work fine and show data with the new override setup?

Reply:
Unfortunately, not all ServiceMonitors can be moved to the Helm charts and all dashboards look fine.

I went with this approach as I expect the Helm chart syntax to rarely change. When these components are removed from the patch functions you would no longer look at them when doing upgrades. Another possiblity would be to revert them back but place a comment that some of them can be overriden in the Helm chart, but are not.